x-pack/filebeat/module/oracle/database_audit/_meta/fields.yml

```diff
@@ -0,0 +1,70 @@
+- name: database_audit
+  type: group
+  release: beta
+  description: >
+    Module for parsing Oracle Database audit logs
+  fields:
+    - name: status
+      type: keyword
+      description: >
+        Database Audit Status.
+
+    - name: sessionid
```

Comment:
This should be `session.id` or even `session_id` if the field is (and will be) single field.